src/kernels/jupyter/launcher/liveshare/hostJupyterServer.ts

```diff
@@ -175,7 +172,7 @@ export class HostJupyterServer implements INotebookServer {
         );
         this.throwIfDisposedOrCancelled(cancelToken);
         const baseUrl = this.connection?.baseUrl || '';
-        this.logRemoteOutput(DataScience.createdNewNotebook(baseUrl));
+        traceVerbose(DataScience.createdNewNotebook(baseUrl));
```

Comment:
One less dependency, we do not need a class to log this data, we can just use `traceVerbose`